### PR TITLE
MAP-1636 cron job recovery

### PR DESCRIPTION
**Pull request by michalnawrockidj**

### CODE REVIEW

**Review by danbenton-mojdt — APPROVED** [review]

**Review by GurnankCheema — DISMISSED** [review]
LGTM

**Comment by jimbali** [comment]
I get this when I run `helm template`:
```
Error: template: use-of-force/templates/job.yaml:6:8: executing "use-of-force/templates/job.yaml" at <include "app.labels" .>: error calling include: template: no template "app.labels" associated with template "gotpl"
```

File: helm_deploy/use-of-force/templates/job.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+apiVersion: batch/v1
```
Comment (marcusaleman) [inline helm_deploy/use-of-force/templates/job.yaml:1]: @michalnawrockidj MAP-1636 ticket is for BVL and the narrative is something differnt than this UOF Cron Job recovery https://dsdmoj.atlassian.net/browse/MAP-1636

**Comment by jimbali** [comment]
You can test generating the templates by running:
```
helm --debug template -f helm_deploy/values-dev.yaml helm_deploy/use-of-force
```

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Comment by marcusaleman** [comment]
@michalnawrockidj MAP-1636 ticket is for BVL and the narrative is something different than this UOF Cron Job recovery https://dsdmoj.atlassian.net/browse/MAP-1636

**Pull request merged**